src/app/static/src/app/store/translations/locales.ts

```diff
@@ -153,6 +157,9 @@ const en: Translations = {
     clickHere: "Click here",
     colorIndicator: "Colour Indicator",
     continue: "continue",
+    copyProjectHeader: "Copying project to a new project",
+    copyToNewProject: "Copy to a new project",
+    copyVersionHeader: "Copying project-version to a new project",
```

Comment:
`project-version` looks a bit weird here, maybe just "Copying version to a new project"? Can be replaced with "Copying {versionId} to a new project" once we have version ids!